app/controllers/organisations_controller.rb

```diff
@@ -1,6 +1,8 @@
 class OrganisationsController < PublicFacingController
   include CacheControlHelper
 
+  CONTENT_NOT_FOUND = Class.new
```

Comment:
It's used as a means to memoize a value that might not exist: https://github.com/alphagov/whitehall/blob/c3c2aa5ddce1521ec48c24a531522f436c043ad2/app/controllers/organisations_controller.rb#L141-L148 as it returning nil won't allow memoize. We had similar confusion when we used this pattern on Pub API too so do wonder if there is a more idiomatic ruby pattern we can use 🤔 - any different approaches to solving this issue in Whitehall that are more consistent?